### ISSUE TITLE
NoSuchMethodError

### ISSUE THREAD

**Issue by nerumo** [jenkinsci/azure-ad-plugin#21]
Since upgrading everything to the latest version I get the following error from the ad plugin:
```
java.io.IOException: java.lang.NoSuchMethodError: com.nimbusds.oauth2.sdk.AccessTokenResponse.<init>
(Lcom/nimbusds/oauth2/sdk/token/AccessToken;Lcom/nimbusds/oauth2/sdk/token/RefreshToken;Ljava/util/Map;)
```
I had to disable the security completely. I tried to set it up again but with the same result.

In my last plugin update were the following plugins involved (all latest version)
```
drwxr-xr-x  21 jenkins jenkins  4.0K Sep 18 16:47 ..
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 pipeline-model-api
drwxr-xr-x 132 jenkins jenkins   20K Sep 18 15:08 .
drwxr-xr-x   6 jenkins jenkins  4.0K Sep 18 15:08 cobertura
drwxr-xr-x   7 jenkins jenkins  4.0K Sep 18 15:08 code-coverage-api
drwxr-xr-x   5 jenkins jenkins  4.0K Sep 18 15:08 cloudbees-folder
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 workflow-cps-global-lib
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 git-parameter
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 junit
drwxr-xr-x   5 jenkins jenkins  4.0K Sep 18 15:08 kubernetes
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 pipeline-stage-tags-metadata
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 azure-commons
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 workflow-basic-steps
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 ssh-slaves
drwxr-xr-x   5 jenkins jenkins  4.0K Sep 18 15:08 azure-app-service
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 script-security
drwxr-xr-x   5 jenkins jenkins  4.0K Sep 18 15:08 workflow-job
drwxr-xr-x   5 jenkins jenkins  4.0K Sep 18 15:08 pipeline-model-definition
drwxr-xr-x   6 jenkins jenkins  4.0K Sep 18 15:08 workflow-cps
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 subversion
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 xunit
drwxr-xr-x   4 jenkins jenkins  4.0K Sep 18 15:08 pipeline-model-extensions
```
Is it related with azure commons?

**Comment by tuthan** [comment]
I have same issue after upgrade today.

**Comment by gavinfish** [comment]
@nerumo @tuthan A new version 0.3.1 has been released to be compatible with 0.2.7 azure commons.

**Comment by nerumo** [comment]
@gavinfish thank you for the fix, I can confirm that the new version fixed the issue.
